x-pack/plugins/apm/public/components/shared/charts/comparison_spark_plot/index.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import {
+  AreaSeries,
+  Chart,
+  CurveType,
+  LineSeries,
+  ScaleType,
+  Settings,
+} from '@elastic/charts';
+import { merge } from 'lodash';
+import { Coordinate } from '../../../../../typings/timeseries';
+import { useChartTheme } from '../../../../../../observability/public';
+import { px, unit } from '../../../../style/variables';
+import { useTheme } from '../../../../hooks/use_theme';
+import { getComparisonChartTheme } from '../../time_comparison/get_time_range_comparison';
+import { Color } from '../spark_plot';
+
+export function ComparisonSparkPlot({
```

Comment:
Do we need this component? Why not have SparkPlot support multiple series?

Reply:
ok, I removed it and added the comparison to the SparkPlot.